apps/explorer/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -43,11 +50,14 @@ function OwnedObject({ id, byAddress }: { id: string; byAddress: boolean }) {
 
         req.then((objects) => {
             let ids: string[];
-            if (objects instanceof Array) {
-                ids = objects.map(({ objectId }) => objectId);
+            if (isPaginatedObjectsResponse(objects)) {
```

Comment:
```suggestion
            if (is(objects, PaginatedObjectsResponse)) {
```
you also need to import `is` from `'@mysten/sui.js'` on line 5. To learn more: https://docs.superstructjs.org/api-reference/core#is